README.md

```diff
@@ -1,3 +1,62 @@
-# juno-sdk
+# Juno SDK
 
-Juno dev SDK
+**Juno SDK** is a development toolkit for interacting with Juno's central infrastructure. It provides convenient models and API methods for handling common tasks like authentication, email management, user management, and project-related operations. This document outlines the core structure of the SDK.
+
+## SDK Structure
+
+### Models
+
+The SDK uses several models to structure the data exchanged between the client and Juno's services. These models represent the input data for API requests and the output data returned by the server.
+
+#### Key Models:
+- **CreateProjectModel**: Defines the structure for creating a new project.
+- **CreateUserModel**: Contains the necessary fields to create a new user (email, name, password).
+- **IssueApiKeyRequest**: Defines the request data for issuing an API key for a project.
+- **IssueApiKeyResponse**: Contains the API key returned when a new API key is issued.
+- **EmailContent**: Represents the content of an email, including its MIME type and body.
+- **EmailRecipient**: Specifies the email and name of the recipient.
+- **EmailSender**: Specifies the email and name of the sender.
+- **ProjectResponse**: Represents the response received when querying project details.
+
+### API Endpoints
+
+The SDK includes API methods for interacting with Juno services. These methods abstract the complexities of making HTTP requests and ensure that data is structured according to Juno's requirements.
+
+#### Core APIs:
+- **AuthApi**: Handles authentication tasks, such as issuing API keys and generating JWT tokens.
+- **EmailApi**: Manages email operations like sending emails and registering domains.
+- **ProjectApi**: Manages project-related tasks, including creating projects and linking users.
+- **UserApi**: Handles user operations, including creating users and managing user roles.
+
+## Usage
+
+The Juno SDK provides a set of APIs for managing various services. Each API is built to simplify interaction with specific Juno infrastructure components. Below is an overview of how to use the main APIs.
+
+### Authentication
+The **AuthApi** is used to handle authentication-related tasks, including issuing API keys and generating JWT tokens for secure access to Juno services.
+
+- **Issue API Key**: Create a new API key for project access. This is necessary for authenticating requests to other services.
```

Comment:
We'll probably want to replace this section with code snippet examples instead. I'd say https://github.com/MONEI/Shopify-api-node could be a good reference README for this